packages/core/src/render3/interfaces/definition.ts

```diff
@@ -131,12 +137,11 @@ export interface DirectiveDef<T> extends BaseDef<T> {
   /**
    * Function to create instances of content queries associated with a given directive.
```

Comment:
Nit: Update docs comment?

```suggestion
   * Function to create and refresh content queries associated with a given directive.
```